tensorflow/python/eager/context.py

```diff
@@ -1204,7 +1205,13 @@ def _compute_gpu_options(self):
     virtual_devices = []
     gpu_index = -1
     memory_growths = set()
-    for dev in self.list_physical_devices("GPU"):
+    gpu_devices = self.list_physical_devices("GPU")
+    pluggable_devices = self._pluggable_devices
+    compatible_devices = gpu_devices
+    for dev in pluggable_devices:
+      if dev not in gpu_devices:
```

Comment:
Could you please give an example of how a device can be in both `gpu_devices` and `pluggable_devices` lists? IIUC, the lists should be disjoint by construction.

Reply:
When PluggableDevice is registered with "GPU" device type(although currently is not supported), the "GPU" would be in gpu_devices as well as pluggable device list.

Reply:
Makes sense. Thank you!